<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
    
    <head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="" />

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc" /><meta name="ncbi_pdid" content="article" /><meta name="ncbi_acc" content="" /><meta name="ncbi_domain" content="wtpa" /><meta name="ncbi_report" content="record" /><meta name="ncbi_type" content="fulltext" /><meta name="ncbi_objectid" content="" /><meta name="ncbi_pcid" content="/articles/PMC2665285/" /><meta name="ncbi_app" content="pmc" />
        <!-- Logger end -->
        
        <title>A CENSUS OF HUMAN CANCER GENES</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css" /><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2665285/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Nature reviews. Cancer" /><meta name="citation_title" content="A CENSUS OF HUMAN CANCER GENES" /><meta name="citation_authors" content="P. Andrew Futreal, Lachlan Coin, Mhairi Marshall, Thomas Down, Timothy Hubbard, Richard Wooster, Nazneen Rahman, Michael R. Stratton" /><meta name="citation_date" content="March 2004" /><meta name="citation_issue" content="3" /><meta name="citation_volume" content="4" /><meta name="citation_firstpage" content="177" /><meta name="citation_doi" content="10.1038/nrc1299" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2665285/?report=abstract" /><meta name="citation_pmid" content="14993899" /><meta name="DC.Title" content="A CENSUS OF HUMAN CANCER GENES" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Europe PMC Funders" /><meta name="DC.Contributor" content="P. Andrew Futreal" /><meta name="DC.Contributor" content="Lachlan Coin" /><meta name="DC.Contributor" content="Mhairi Marshall" /><meta name="DC.Contributor" content="Thomas Down" /><meta name="DC.Contributor" content="Timothy Hubbard" /><meta name="DC.Contributor" content="Richard Wooster" /><meta name="DC.Contributor" content="Nazneen Rahman" /><meta name="DC.Contributor" content="Michael R. Stratton" /><meta name="DC.Date" content="2004 Mar" /><meta name="DC.Identifier" content="10.1038/nrc1299" /><meta name="DC.Language" content="en" /><meta property="og:title" content="A CENSUS OF HUMAN CANCER GENES" /><meta property="og:type" content="article" /><meta property="og:description" content="A central aim of cancer research has been to identify the mutated genes that are causally implicated in oncogenesis (‘cancer genes’). After two decades of searching, how many have been identified and how do they compare to the complete ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2665285/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png" /><meta name="twitter:card" content="summary" /><meta name="twitter:site" content="@ncbi" /><link rel="stylesheet" href="/corehtml/pmc/css/3.6.2/pmc.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.6.2/pmc_extras_prnt.min.css" type="text/css" media="print" /><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'/pmc/utils/ctxp/'" /><script type="text/javascript" src="/corehtml/pmc/ctxp/jquery.citationexporter.js">//</script><link rel="stylesheet" href="/corehtml/pmc/ctxp/citationexporter.css" type="text/css" /><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript" src="/corehtml/pmc/js/large-obj-scrollbars.min.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content="" /><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2665285/epub/" /><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2665285/pdf/ukmss-2560.pdf" />

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A20C684395CE100000000004D0045" />
<meta name='referrer' content='origin-when-cross-origin'/><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4144485/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/4120508/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4144485/css/3411343/3882866/3929378.css" media="print" /></head>
    <body class="article">
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4144485/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="toolkitbookgh">ToolKitBookgh</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC2665285</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=nihms" target="pmc_ext" /><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=nihms" target="pmc_ext" /><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=nihms" target="pmc_ext" /></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="small"><div class="inline_block eight_col va_top"><div><span id="pmcmata">Nat Rev Cancer</span>. Author manuscript; available in PMC 2009 Apr 5.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=14993899" target="pmc_ext" ref="reftype=publisher&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access&amp;rendering-type=nihms"><span class="cit">Nat Rev Cancer. 2004 Mar; 4(3): 177–183. </span></a></div> <span class="doi">doi:  <a href="//dx.doi.org/10.1038%2Fnrc1299" target="pmc_ext" ref="reftype=other&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI&amp;rendering-type=nihms">10.1038/nrc1299</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2665285</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>UKMS2560</span></div></div></div></div></div><h1 class="content-title">A CENSUS OF HUMAN CANCER GENES</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Futreal%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=14993899">P. Andrew Futreal</a>,<sup>*</sup> <a href="/pubmed/?term=Coin%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=14993899">Lachlan Coin</a>,<sup>*</sup> <a href="/pubmed/?term=Marshall%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=14993899">Mhairi Marshall</a>,<sup>*</sup> <a href="/pubmed/?term=Down%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=14993899">Thomas Down</a>,<sup>*</sup> <a href="/pubmed/?term=Hubbard%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=14993899">Timothy Hubbard</a>,<sup>*</sup> <a href="/pubmed/?term=Wooster%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=14993899">Richard Wooster</a>,<sup>*</sup> <a href="/pubmed/?term=Rahman%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=14993899">Nazneen Rahman</a>,<sup>&#x02021;</sup> and  <a href="/pubmed/?term=Stratton%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=14993899">Michael R. Stratton</a><sup>*</sup><sup>&#x02021;</sup></div></div><div class="fm-panel small half_rhythm"><div class="fm-authors-info fm-panel hide half_rhythm" id="id1025201_ai" style="display:none"><div class="fm-affl" lang="en"><sup>*</sup>Cancer Genome Project, Human Genome Analysis Group and Pfam Group, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton Cambs, CB10 1SA, UK. </div><div class="fm-affl" lang="en"><sup>&#x02021;</sup>Section of Cancer Genetics, Institute of Cancer Research, 15 Cotswold Rd, Sutton, Surrey SM2 5NG, UK. </div><div id="CR1">Correspondence to M.R.S. e-mail: <a href="mailto:dev@null" data-email="ku.ca.regnas@srm" class="oemail">ku.ca.regnas@srm</a></div></div><div class="togglers"><a href="#" class="pmctoggle" rid="id1025201_ai">Author information <span>►</span></a> <a href="#" class="pmctoggle" rid="id1025201_cpl">Copyright and License information <span>►</span></a></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="fm-cpl-info fm-panel hide half_rhythm" id="id1025201_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a>  and <a href="/pmc/about/disclaimer/">Disclaimer</a></div></div></div><div id="pmclinksbox" class="links-box fm-panel whole_rhythm"><div class="small"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=14993899" target="pmc_ext" ref="reftype=publisher&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access&amp;rendering-type=nihms">Nat Rev Cancer</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2665285/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="__abstractid1032146" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="__abstractid1032146title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">A central aim of cancer research has been to identify the mutated genes that are causally implicated in oncogenesis (&#x02018;cancer genes&#x02019;). After two decades of searching, how many have been identified and how do they compare to the complete gene set that has been revealed by the human genome sequence? We have conducted a &#x02018;census&#x02019; of cancer genes that indicates that mutations in more than 1% of genes contribute to human cancer. The census illustrates striking features in the types of sequence alteration, cancer classes in which oncogenic mutations have been identified and protein domains that are encoded by cancer genes.</p></div></div><div id="__bodyid1032156" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">Numerous alterations in DNA sequence underlie the development of every neoplasm. These sequence variants can be transmitted through the germline and result in susceptibility to cancer, or can arise by somatic mutation. A central aim of cancer research has been to identify the mutated genes that are causally implicated in oncogenesis (referred to here as &#x02018;cancer genes&#x02019;). Since the advent of recombinant-DNA technology more than two decades ago, there has been considerable success in this enterprise. Following the first report of a somatic mutation in a human cancer gene &#x02014; the conversion of amino-acid 12 of HRAS from glycine to valine in the human bladder carcinoma cell line T24/EJ<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278132">1</a> &#x02014; a substantial number of human cancer genes have been identified, and their biological properties have been assiduously investigated. The proteins that are encoded by cancer genes normally regulate cell proliferation, cell differentiation and cell death. Mutations underlying oncogenesis also occur in genes that mediate DNA-repair processes<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278113">3</a><sup>-</sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278123">5</a>. These result in an increased rate of somatic mutation, which, in turn, might increase the likelihood of a growth-control gene being mutated. We have compiled a list of cancer genes from the published literature. The census illustrates several striking features with respect to the types of sequence alteration, cancer classes in which oncogenic mutations have been identified and protein domains that are encoded by cancer genes.</p></div><div id="S1" class="tsec sec"><h2 class="head no_bottom_margin" id="S1title">Criteria for inclusion in the cancer-gene census</h2><p id="P3" class="p p-first">We have exclusively listed genes in which mutations that are causally implicated in oncogenesis have been reported (see supplementary information S1 (table)) . Most cancer genes have been identified and initially reported on the basis of genetic evidence (that is, the presence of somatic or germline mutations) and without biological information supporting the oncogenic effects of the mutations. The underlying rationale for interpreting a mutated gene as causal in cancer development is that the number and pattern of mutations in the gene are highly unlikely to be attributable to chance. So, in the absence of alternative plausible explanations, the mutations are likely to have been selected because they confer a growth advantage on the cell population from which the cancer has developed. There often exists additional biological information supporting the role of a mutated gene in oncogenesis. However, if the genetic data is convincing (see below), we have regarded it as sufficient on its own for inclusion of the gene in the census (<a href="#BX2" rid="BX2" class=" boxed-text">BOX 1</a>).</p><div class="boxed-text-box whole_rhythm hide-overflow" id="BX2"><h3>Box 1</h3><div class="caption"><h4>Genes that have been included in the cancer-gene census</h4></div><div id="P23"><p>Genes with the following types of mutations have been included:
</p><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
--><li><div>Base substitutions that lead to missense amino-acid changes, nonsense changes and alterations in the well-conserved positions of splice sites.</div></li><li><div>Insertions or deletions in coding sequences or splice sites that might cause in-frame or frameshifting alterations in the protein.</div></li><li><div>Rearrangements because of chromosomal translocations that lead to chimeric transcripts or to deregulation of genes through apposition to novel promoter or enhancer regions.</div></li><li><div>Copy-number increases and decreases.</div></li></ul></div></div><p id="P4">Genes have been included in the census if there exist at least two independent reports showing mutations in primary patient material. In considering somatic mutations for inclusion in the census, we included only genes for which there was evidence of the somatic origin of at least a subset of mutations, based on analysis of normal tissue from the same individuals. In practice, this is most important for the interpretation of putative somatic base substitutions and small insertions/deletions, because these are also common as germline sequence variants. By contrast, translocations and copy-number changes are relatively uncommon as germline variants. Therefore, we have included some cancer genes that are altered by translocation or copy-number change for which definitive evidence of the somatic origin of their mutations is unavailable.</p><div class="boxed-text-box whole_rhythm hide-overflow" id="BX1"><div class="caption"><h3>Summary</h3></div><div id="P5"><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
--><li><div>We have conducted a census from the literature of genes that are mutated and causally implicated in cancer development (&#x02018;cancer genes&#x02019;).</div></li><li><div>So far, 291 cancer genes have been reported, more than 1% of all the genes in the human genome.</div></li><li><div>90% of cancer genes show somatic mutations in cancer, 20% show germline mutations and 10% show both.</div></li><li><div>The most common mutation class among the known cancer genes is a chromosomal translocation that creates a chimeric gene or apposes a gene to the regulatory elements of another gene.</div></li><li><div>Many more cancer genes have been found in leukaemias, lymphomas and sarcomas than in other types of cancer, despite the fact that they represent only 10% of human cancer. These genes are usually altered by chromosomal translocation.</div></li><li><div>The most common domain that is encoded by cancer genes is the protein kinase. Several domains that are involved in DNA binding and transcriptional regulation are common in proteins that are encoded by cancer genes.</div></li></ul></div></div><p id="P12">The census does not include genes that are only expressed at altered levels in cancer cells, without any mutation in DNA. Although alterations in expression might be part of the reconfiguration of cell biology following the acquisition of a mutation in a cancer gene, they are usually not the primary change. Indeed, in some cases they might be consequences of the neoplastic phenotype rather than determinants of it.</p><p id="P13">The list also does not include genes in which changes in methylation of CpG dinucleotides within promoter regions have been the only reported abnormalities. Many such alterations are clearly associated with altered transcriptional regulation and, in some cases (for example, in the promoters of <em>CDKN2A</em> or <em>MLH1</em>), are likely to cause neoplastic transformation<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278130">6</a><sup>,</sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278135">7</a>. Methylation of promoter regions is, however, relatively plastic. It can change in association with age or <em>in vitro</em> culture, is widespread in some cancer genomes and can occur in C<span class="small-caps">P</span>G ISLANDS without any known consequences for transcriptional regulation<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278112">8</a>. It is therefore problematic to identify genes in which alterations in promoter methyation, without mutation, cause neoplastic change, so this form of genetic modification has not been included in our database.</p><p id="P18" class="p p-last">It is likely that some cancer genes have inadvertently been missed, and opinions might differ about some genes that have been excluded and included. We are keen to be informed of these omissions and controversial cases in order to evaluate their status and will regularly update this list on the Wellcome Trust Sanger Institute web site (see <a href="#APP2" rid="APP2" class=" app">online links box</a>).</p></div><div id="S2" class="tsec sec"><h2 class="head no_bottom_margin" id="S2title">Problems encountered in compiling the census</h2><p id="P19" class="p p-first">We have attempted to be comprehensive in compilation of the cancer-gene list, but have also been conservative in our criteria for inclusion. Several genes that were considered were ultimately excluded. Below, we have reviewed some common problems that we encountered in determining whether a gene should be included as a cancer gene.</p><div id="S3" class="sec"><h3>Genes with few reported mutations</h3><p id="P20" class="p p-first-last">All classes of mutation occur as more or less random processes in both normal and neoplastic cells. Indeed, it is likely that most somatic mutations do not confer a clonal growth advantage. So, cancer genomes carry several &#x02018;PASSENGER&#x02019; OR &#x02018;BYSTANDER&#x02019; MUTATIONS, in addition to the mutations that cause the neoplastic phenotype. Unfortunately, the prevalence of passenger mutations in the genomes of most cancer types is not known. Therefore, if somatic mutations in a putative cancer gene are found in a very small proportion of tumours, it is difficult to exclude the possibility that they represent chance clustering of passenger alterations. Consequently, we have excluded genes in which fewer than five unambiguous somatic mutations have been reported in primary neoplasms.</p></div><div id="S4" class="sec"><h3>Cancers with large numbers of mutations</h3><p id="P21" class="p p-first">A particular problem arises in cancer cells that have genomes with a high prevalence of somatic mutations. For example, cancers with mismatch-repair (MMR) gene defects carry tens of thousands of small insertions and deletions in short tandem repeats (known as microsatellites)<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278111">9</a>. Most of these occur in intronic or intergenic DNA and are, therefore, almost certainly passenger mutations. However, several hundred are in coding sequences and will often result in translational frameshifts, with the consequence of premature protein termination or nonsense-mediated RNA decay. It is highly likely that a small percentage of these are causally involved in oncogenesis. It is equally probable, however, that most are not. In these circumstances, distinguishing mutated cancer genes from genes with clusters of passenger mutations is particularly problematic.</p><p id="P22" class="p">Several genes with small insertions and deletions of coding microsatellites/mononucleotide repeats that lead to protein truncation in MMR-deficient tumours have been proposed as cancer genes<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278125">9</a>. Attempts have been made to distinguish mutations that confer a clonal growth advantage from bystander mutations in MMR-deficient cells<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278137">10</a>. Transforming growth factor-<em>&#x003b2;</em> type II receptor (<em>TGF-&#x003b2;RII</em>), which mediates the effects of the growth-suppressing ligand <em>TGF-&#x003b2;</em>, is one example of a potential cancer gene that was discovered in MMR-deficient cells. Mutations in <em>TGF-&#x003b2;RII</em> are believed to be more than simple bystander mutations because of the biological function of its gene product, the high frequency of biallelic mutations in MMR-deficient cells and the presence of some biologically relevant somatic mutations in MMR-proficient cancers<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278119">11</a>. However, the distinction is still unclear for most genes that have microsatellite instability in their coding sequence. Therefore, genes in which somatic mutations occur exclusively or predominantly in MMR-deficient cells have been excluded from the cancer-gene census.</p></div><div id="S5" class="sec"><h3>Putative cancer genes with particularly high mutation rates in cancer cells</h3><p id="P28" class="p p-first-last">A similar problem arises when there is an increased mutation rate restricted to a small region of the genome. For example, the fragile histidine triad gene (<em>FHIT</em>) and WW-domain-containing oxidoreductase (<em>WWOX</em>) genes each straddle a fragile site (<em>FRA3B</em> and <em>FRA16D</em>, respectively<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278109">12</a><sup>,</sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278134">13</a>). Fragile sites are genomic regions that are associated with a high frequency of chromosome breakage, and many were originally identified through chemical stress of normal lymphocytes<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278107">14</a>. The intrinsic fragility of these regions is frequently expressed in cancer cells as rearrangements and deletions, mutational patterns similar to those found in certain recessive cancer genes (for example, <em>CDKN2A</em> and <em>PTEN</em>&#x0200b;<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278139">15</a><sup>,</sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278144">16</a>). Therefore, based simply on mutation clustering and pattern, it is not clear whether <em>FHIT</em> and <em>WWOX</em> are recessive cancer genes (and therefore causally implicated in oncogenesis) or fragile sites that are frequently rearranged in cancer cells because of defects in the DNA-repair process, or both. Most well-established recessive cancer genes also contain somatic base substitutions and small insertions/deletions that cause protein truncation. These are uncommon in <em>FHIT</em> and <em>WWOX</em>. So, although there are additional biological data that might support a role for genes such as <em>FHIT</em> and <em>WWOX</em> in oncogenesis, genes overlying fragile sites are not included in this census of cancer genes.</p></div><div id="S6" class="sec"><h3>Mutations that encompass many genes</h3><p id="P29" class="p p-first-last">Most classes of mutations (such as base substitutions) only affect a single gene, or, at most, two genes (such as reciprocal chromosomal translocations). However, copy-number changes, such as gene amplification, can affect several megabases of DNA and encompass many genes. Therefore, solely on the basis of genetic evidence, it is not always clear which gene is the crucial target of the amplification, and it is conceivable that, in some cases, there is more than one target. Additional evidence that might be invoked in assessing which gene(s) on an amplicon mediates oncogenesis includes increased expression levels in cancer cells and functional effects. Chromosome defects, such as trisomy, that involve even larger regions of the genome pose even greater problems, as they might alter the copy number of thousands of genes. We have therefore only included in the census amplified genes for which there is reasonable consensus that the cancer-causing gene has been identified, although even these might be controversial. Many amplified genes that have not been included in the cancer-gene census clearly exist and might be included in the future.</p></div><div id="S7" class="sec sec-last"><h3>Low-penetrance cancer-susceptibility genes</h3><p id="P30" class="p p-first-last">There is usually little ambiguity in the identification of mutated genes that are responsible for high-penetrance (high-risk) cancer-susceptibility syndromes or of mutated genes that are associated with characteristic non-neoplastic manifestations in addition to cancer predisposition. However, germline variants of many genes have recently been proposed as low-penetrance (low-risk) cancer-susceptibility alleles without additional non-neoplastic features. In some cases &#x02014; for example, for the variants <em>APC</em>*I1307K<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278133">17</a> and <em>CHK2</em>*1100delC<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278117">18</a> &#x02014; the evidence for this effect is strong. For most genes, however, the evidence is statistically weak and often conflicting. Therefore, for purposes of clarity, we have not included this type of cancer-susceptibility gene in the current census.</p></div></div><div id="S8" class="tsec sec"><h2 class="head no_bottom_margin" id="S8title">Accessory information in the cancer-gene census</h2><p id="P31" class="p p-first">To facilitate further analyses and to illustrate some striking patterns, we have appended additional information to the list of cancer genes &#x02014; see supplementary information S1 (table). Fields in the census include the gene name, symbol, LocusLink number, protein accession number and chromosomal location. Further fields indicate whether the gene is somatically mutated in cancer or mutated in the germline predisposing to cancer (or both). Cancer types and syndrome names that are associated with somatic and germline mutations in each gene have been included. These lists are not exhaustive and are intended to present a general picture of the cancer types that are associated with individual mutated cancer genes. So, they include the main classes of neoplasm that have been reported, but omit tumour types that infrequently show mutations.</p><p id="P32" class="p p-last">We have also included a field in which neoplasms are classified into four groups: leukaemias/lymphomas, tumours of mesenchymal origin (sarcomas and benign mesenchymal tumours), epithelial tumours and others. This classification is arbitrary, but it facilitates certain types of analysis and detection of some notable patterns. Cancer genes have also been classified according to whether, at the cellular level, mutations are dominant (that is, a single mutated allele is sufficient to contribute to oncogenesis) or recessive (that is, both alleles need to be mutated &#x02014; sometimes termed &#x02018;tumour-suppressor genes&#x02019;). We recognize that this classification is sometimes presumptive and occasionally an oversimplification. Finally, there is a field indicating the types of mutation that are observed in each gene. For recessive cancer genes, two mutations will usually be present in a single tumour sample and these often involve two of the categories of mutation listed. The proportions of cancer genes that are mutated in somatic or germline cells (<a href="/pmc/articles/PMC2665285/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>FIG. 1</span></a>), that are mutated by translocation or by other types of mutation (<a href="/pmc/articles/PMC2665285/figure/F2/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>FIG. 2</span></a>), that act in a dominant or recessive manner (<a href="/pmc/articles/PMC2665285/figure/F3/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>FIG. 3</span></a>), and that are mutated in the various classes of neoplasm (<a href="/pmc/articles/PMC2665285/figure/F4/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>FIG. 4</span></a>) are shown.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a class="icnblk_img figpopup" href="/pmc/articles/PMC2665285/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><img src="/pmc/articles/PMC2665285/bin/ukmss-2560-f0001.gif" class="small-thumb" alt="Figure 1" title="Figure 1" src-large="/pmc/articles/PMC2665285/bin/ukmss-2560-f0001.jpg" /></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2665285/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a4--><div><span><!--caption a8--><strong>Mutation types in human cancer</strong></span></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a class="icnblk_img figpopup" href="/pmc/articles/PMC2665285/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><img src="/pmc/articles/PMC2665285/bin/ukmss-2560-f0002.gif" class="small-thumb" alt="Figure 2" title="Figure 2" src-large="/pmc/articles/PMC2665285/bin/ukmss-2560-f0002.jpg" /></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2665285/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a4--><div><span><!--caption a8--><strong>Proportion of translocated genes in human cancer</strong></span></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a class="icnblk_img figpopup" href="/pmc/articles/PMC2665285/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><img src="/pmc/articles/PMC2665285/bin/ukmss-2560-f0003.gif" class="small-thumb" alt="Figure 3" title="Figure 3" src-large="/pmc/articles/PMC2665285/bin/ukmss-2560-f0003.jpg" /></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2665285/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a4--><div><span><!--caption a8--><strong>Dominant and recessive mutations in human cancer</strong></span></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock ten_col whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a class="icnblk_img figpopup" href="/pmc/articles/PMC2665285/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><img src="/pmc/articles/PMC2665285/bin/ukmss-2560-f0004.gif" class="small-thumb" alt="Figure 4" title="Figure 4" src-large="/pmc/articles/PMC2665285/bin/ukmss-2560-f0004.jpg" /></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC2665285/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a4--><div><span><!--caption a8--><strong>Proportion of genes associated with different tumour types</strong></span></div></div></div></div><div id="S9" class="tsec sec"><h2 class="head no_bottom_margin" id="S9title">General features of cancer genes</h2><p id="P33" class="p p-first">Mutations in at least 291 human genes are causally implicated in oncogenesis (see supplementary information S1 (table)). All encode proteins. No mutated RNA genes have yet been shown to be involved in oncogenesis. Assuming that there are approximately 25,000 coding genes in the human genome &#x02014; as indicated on the Ensembl Human Genome Browser web page (see <a href="#APP2" rid="APP2" class=" app">online links box</a>) &#x02014; it seems that mutations in more than 1% genes are so far known to be involved in cancer pathogenesis.</p><p id="P34">Most were identified by positional cloning without any previous hypothesis of biological function. By this strategy, the cancer gene is first localized to a small part of the genome and, subsequently, genes within the delimited interval are screened for mutations. The primary positional clues have been diverse and include chromosomal rearrangements that are visible in metaphase spreads of cancer cells, DNA copy-number changes in cancer cells, (which are detected by various molecular approaches) and, for cancer susceptibility genes, genetic-linkage analyses of families with several cases of cancer.</p><p id="P35">A small number of cancer genes have been detected through biological assays. Most notable among these has been the NIH-3T3 transformation assay, in which total human DNA is introduced into a line of mouse fibroblasts, and cells that incorporate certain classes of mutated human cancer genes acquire the transformed phenotype<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278115">19</a><sup>,</sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278104">20</a>. Mutations in the remainder of human cancer genes have been identified through analysis of plausible candidates based on known biological features of cancer cells. However, these constitute a small minority of known cancer genes.</p><p id="P36">Approximately 90% of cancer genes show somatic mutations and 20% show germline mutations. Both somatic and germline mutations have been reported in 10% of all cancer genes (<a href="/pmc/articles/PMC2665285/figure/F1/" target="figure" class="fig-table-link fig figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>FIG. 1</span></a>). In general, the spectrum of neoplasms that are associated with germline mutations in a particular gene is similar to that reported with somatic mutations. There are, however, several notable exceptions to this rule. For example, somatic mutations in <em>TP53</em> are found in more than half of colorectal cancers, yet germline mutations do not apparently cause a predisposition to colorectal cancer<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278128">21</a><sup>,</sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278141">22</a>. Similarly, germline mutations in <em>STK11</em> are associated with a predisposition to hamartomas of the gastrointestinal tract and to colorectal, pancreatic and ovarian neoplasms (the Peutz&#x02013;Jegher syndrome)<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278116">23</a>. However, somatic mutations of <em>STK11</em> have only been found in lung adenocarcinomas, which are not usually considered to be components of the Peutz&#x02013;Jegher syndrome<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278127">24</a>. Several genes with germline mutations that cause cancer predisposition show very few, if any, somatic mutations in sporadic cancers of the same type, such as <em>BRCA1</em> and <em>BRCA2</em> in breast cancer<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278105">25</a><sup>,</sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278129">26</a>. The reasons for differences between the tumour spectrum that is associated with somatic mutations and the spectrum that is associated with germline mutations are generally unknown.</p><p id="P37">The most common class of somatic mutation that is registered in the cancer-gene census involves chromosomal translocations that result in a chimeric transcript or apposition of one gene to the regulatory regions of another gene &#x02014; usually immunoglobulin or T-cell-receptor genes. This mutation type is common in leukaemias, lymphomas and mesenchymal tumours. However, several examples have now been reported among epithelial neoplasms, including thyroid papillary carcinoma (<em>RET</em> and <em>NTRK1</em>, both with several partners), thyroid follicular carcinoma (<em>PAX8</em> and <em>PPAR&#x003b3;</em>), renal papillary carcinoma (<em>PRCC</em> and <em>TFE3</em>) and breast secretory carcinomas (<em>ETV6</em> and <em>NTR3</em>)<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278143">27</a><sup>-</sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278106">30</a>. Because two genes are structurally rearranged in each chromosomal translocation, the number of translocated cancer genes, compared with other types of mutated cancer gene, is exaggerated in the census. Moreover, certain genes, such as <em>MLL</em> (mixed-lineage leukaemia), are highly promiscuous and form chimeric transcripts with a large number of partners<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278140">31</a>. As a consequence of this ability to form chimeric genes with more than one partner, &#x02018;networks&#x02019; of translocation partners can be discerned. For example, <em>MLL</em> can form a chimeric cancer gene by chromosomal translocation with CREB-binding protein (<em>CREBBP</em>; also known as <em>CBP</em>). In addition to <em>MLL</em>, <em>CBP</em> can also form a chimeric cancer gene with <em>RUNXBP2</em> (also known as <em>ZNF220</em>). <em>RUNXBP2</em>, in turn, can form a chimeric cancer gene with <em>EP300</em>, and this gene can from a chimeric cancer gene with <em>MLL</em>.</p><p id="P38">For some genes, several different types of mutations have been associated with cancer. Notably, many of the different classes of mutation are found in most of the recessive cancer genes, because the result is usually inactivation of the encoded protein, and there are many mutation types that can achieve this end. However, for dominantly acting cancer genes, in which the encoded protein is usually activated, the patterns of mutation are much more restricted, so only one class of mutation is usually found.</p><p id="P39">Indeed, if more than one class of mutation occurs in a dominantly acting cancer gene, each type of mutation might be associated with a particular type of cancer. For example, two classes of mutation result in constitutive activation of the RET kinase &#x02014; chromosomal translocations and base substitutions that lead to missense amino-acid changes. Rearrangements of RET are found in papillary thyroid carcinoma, whereas base substitutions are found in medullary thyroid cancer<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278136">32</a>. Occasionally, a single allele of a cancer gene might require more than one mutation for full biological effect. For example, in-frame deletions within the extracellular domain of EGFR are common in malignant gliomas<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278118">33</a>. Usually, the rearranged EGFR allele is also amplified. Similar increases in copy number occur on alleles mutated by base substitutions, for example MET in renal papillary carcinoma<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278138">34</a>.</p><p id="P42">More than 70% of cancer genes with somatic mutations in the census are associated with leukaemias, lymphomas and mesenchymal tumours, even though these account for less than 10% of human cancer incidence. Why have so many more genes been associated with these relatively rare tumour types than with the more common epithelial cancers? Some of this imbalance is attributable to the number of genes that are subject to chromosomal translocations in leukaemias, lymphomas and mesenchymal tumours, with the attendant double counting of genes (see above). Another explanation is that it might have been easier, in the past, to identify cancer genes by studying leukaemias, lymphomas and mesenchymal neoplasms than by studying epithelial cancers. If this interpretation is correct, it indicates that many more cancer genes remain to be identified in association with common epithelial cancers. Alternatively, there might be fundamental biological differences between the group of leukaemias, lymphomas and mesenchymal tumours and the group of common epithelial neoplasms, such that the repertoire of mutated genes that is necessary to generate common epithelial neoplasms is more restricted.</p><p id="P43">90% of somatic mutations in cancer genes are dominant at the cellular level. Again, this is predominantly determined by the frequency of chromosomal translocations in leukaemias, lymphomas and mesenchymal tumours, almost all of which act in this way. If cancer genes that contribute to oncogenesis by chromosomal translocation are excluded, equal numbers of somatically mutated cancer genes are dominant and recessive at the cellular level. By contrast, 90% of germline mutations that result in cancer predisposition are recessive at the cellular level, presumably because many cancer-causing mutations that might act in a dominant fashion would cause embryonic lethality.</p><p id="P44" class="p p-last">There is at least one example of a cancer gene that is able to act through both dominant and recessive mechanisms. PRKAR1A &#x02014; a regulatory subunit of protein kinase A &#x02014; is a translocation partner with RET. This translocation activates the RET kinase in papillary carcinoma of the thyroid<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278122">35</a>. By contrast, germline mutations that inactivate <em>PRKAR1A</em> are associated with predisposition to Carney complex, a rare syndrome that is characterized by endocrine tumours and MYXOMA of the heart<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278110">36</a>.</p></div><div id="S10" class="tsec sec"><h2 class="head no_bottom_margin" id="S10title">Biological features of cancer proteins</h2><p id="P45" class="p p-first">There are now over 2,600 classes of protein domain reported on Pfam (see <a href="#APP2" rid="APP2" class=" app">online links box</a>) &#x02014; a manually curated database of protein-domain families &#x02014; that are encoded by genes in the human genome<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278108">37</a>. Of these, we found 221 in proteins that are encoded by cancer genes. We compared Pfam domains that are encoded by cancer genes to Pfam domains that are encoded by the complete human gene set (see supplementary information S2 (table)). The analysis has also been applied to subgroups of the cancer-gene list (see supplementary information S3,S4 (tables)). Compared with their representation in proteins that are encoded by the complete human-gene set, at least 11 Pfam domains are clearly over-represented among proteins that are encoded by cancer genes (see supplementary information S2 (table)). These include the protein-kinase, bromodomain, helix&#x02013;loop&#x02013;helix (HLH), homeobox, carboxy-terminal DNA-binding (ETS), PAX, prolyl hydroxylase, MMR, HATPase_c, MYC amino-terminal and AF-4 domains.</p><p id="P46">The most commonly represented Pfam domain that is encoded by cancer genes is the protein kinase, and this is also the domain for which there is strongest evidence of over-representation &#x02014; there are 27 cancer genes in the census that encode protein-kinase domains, compared with the 6.3 that would be expected in a random selection of the same number of genes from the complete set of human genes (see supplementary information S2 (table)). Some over-representation of protein-kinase domains might be attributable to ascertainment bias. Protein kinases have long been implicated in oncogenesis and, consequently, mutations in some have been identified because they were examined as plausible candidates. However, mutations in most protein kinases in the cancer-gene census were identified through positional-cloning approaches, so ascertainment bias is unlikely to account completely for their over-representation.</p><p id="P47">Most of the protein kinases in the cancer-gene census show somatic mutations in cancer, but there are several in which germline mutations cause predisposition to neoplasia, including <em>MET</em>, <em>KIT</em>, <em>STK11</em>, and <em>CDK4</em>. Most cancer genes that encode protein kinases contain activating mutations and are dominant at the cellular level. However, a minority act in a recessive manner at the cellular level. These include <em>ATM</em>, <em>STK11</em>, and <em>BMPR1A</em>, which are all inactivated by mutations<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278114">38</a><sup>,</sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278124">23</a><sup>,</sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278142">39</a>. Mutated protein kinases are particularly strongly over-represented among epithelial neoplasms, but are also found in leukaemias, lymphomas and mesenchymal tumours. Dominantly-acting mutated protein kinases are activated by diverse classes of mutations, including gene amplification, base substitution, in-frame large insertions and deletions (for example, <em>FLT3</em> and <em>EGFR</em>, respectively)<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278121">33</a><sup>,</sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278126">40</a>, in-frame small deletions (for example, <em>KIT</em>&#x0200b;, <em>PDGFRA</em>)<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278120">41</a><sup>,</sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_292278131">42</a> and chromosomal translocation. Tyrosine kinases and serine/threonine kinases are both represented in the cancer-gene census. However, tyrosine kinases are over-represented compared with serine/threonine kinases, accounting for approximately one quarter of all the known protein kinases and two-thirds of the protein kinases that are encoded by cancer genes. Interestingly, phosphatases are not prominent in the cancer-gene census &#x02014; one tyrosine phosphatase is listed, approximately the expected number in a random selection of genes from the complete set of human genes.</p><p id="P48">After protein kinases, the most frequently over-represented Pfam domains are those that broadly constitute components of proteins that are implicated in transcriptional regulation. These include HLH, ETS, PAX, homeobox, MYC N-terminal, bromodomain, AF-4 and PHD domains. Many of these (for example, PAX, ETS, AF-4, HLH, bromodomain and MYC N-terminal) are over-represented tenfold or more in the cancer-gene census, compared with the numbers expected from a random selection of human proteins. In contrast to the protein-kinase domain, most domains that are involved in transcriptional regulation are encoded by cancer genes activated by chromosomal translocations in leukaemias, lymphomas and mesenchymal tumours.</p><p id="P49">The final group of domains that are clearly over-represented among cancer genes is associated with DNA maintenance and repair (MMR and HATPase_c domains). Mutated cancer genes encoding these domains generally act in a recessive manner at the cellular level, are inactivated during oncogenesis (resulting in increased somatic mutation rates) and often have germline mutations that result in cancer predisposition. Indeed, a substantial proportion of germline-mutated genes that cause cancer predisposition are involved in DNA maintenance and repair.</p><p id="P50">Other Pfam domains that are frequently encoded by cancer genes are not necessarily over-represented in the cancer-gene census. For example, ten cancer genes encode C2H2 zinc-finger domains (which are implicated in DNA binding and transcriptional regulation). However, the C2H2 zinc finger is a common motif and this is the number that would be expected based on a random selection of human genes. Certain Pfam domains are, however, under-represented among cancer genes. For example, only one cancer gene encodes a rhodopsin-like seven-transmembrane domain, compared with nine expected (see supplementary information S2 (table)). Rhodopsin-like seven-transmembrane domains form a large class of G-protein-coupled receptors (GPCRs) that respond to a wide variety of signals. Their under-representation among cancer genes is perhaps surprising, given the over-representation of protein kinases, as both groups of proteins are involved in signal transduction. However, the results indicate that the normal metabolic connections of many GPCRs do not substantially influence the processes of cell proliferation, differentiation and death that underlie neoplastic change.</p><p id="P51" class="p p-last">This census provides a detailed view of cancer-associated genes, the mutations that contribute to tumorigenesis and the functional consequences of the resulting structural abnormalities. Some clear patterns and questions emerge. Even with our relatively conservative inclusion criteria, we find more than 1% of genes in the human genome are involved in oncogenesis. The total number of human cancer genes remains a matter for speculation. For most individual adult epithelial cancers, it is not possible at present to identify the four to seven somatically mutated cancer genes that are usually proposed to be necessary (as a minimum) for cancer development. There also seem to be more cancer genes on the way to being identified by conventional strategies. For example, there are several recurrent copy-number abnormalities found in human cancer for which the target gene has yet to be definitively identified, and there could be many genes with germline sequence variants that confer a small additional risk of cancer (low-penetrance cancer-susceptibility genes). Moreover, positional-cloning strategies (in the past, the most influential approaches to cancer-gene identification) might have completely missed many mutated cancer genes simply because they do not yield informative positional cues. Finally, the full role of promoter methylation in cancer and the number of genes that contribute to oncogenesis when modified in this way is yet to be clarified. So, it is plausible (although unproven), that many more cancer genes remain to be identified. The finished human genome sequence now offers new opportunities for identifying cancer genes. It will be interesting to observe if the current patterns persist, or whether they predominantly reflect the technical opportunities and constraints that have prevailed in the past twenty years of cancer-gene identification.</p></div><div id="G1" class="tsec sec"><h2 class="head no_bottom_margin" id="G1title">Glossary</h2><div class="bk-sec"><div><table class="default-table glossary"><tbody><tr><td>C<span class="small-caps">P</span>G ISLANDS</td><td>GC-rich areas of the genome, usually of the order of a kilobase in size, often in and around the 5&#x02032; regions of genes, which retain an unusually high number of CpG dinucleotides.</td></tr><tr><td>PASSENGER OR BYSTANDER MUTATIONS</td><td>Somatic mutations that are found in cancer cells that are not involved in generating the neoplastic phenotype.</td></tr><tr><td>MYXOMA</td><td>A rare type of tumour that is usually composed of sparse mesenchymal cells interspersed amid large amounts of intercellular material.</td></tr></tbody></table></div></div></div><div id="__app-groupid1062037" class="tsec sec"><h2 class="head no_bottom_margin" id="__app-groupid1062037title">Online Links</h2><div class="bk-sec"><div id="APP1" class="sec sec-first"><h3>DATABASES</h3><p id="P53">The following terms in this article are linked online to:</p><p id="P54"><strong>Cancer.gov</strong>: <a href="http://cancer.gov/" ref="reftype=extlink&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=nihms" target="pmc_ext">http://cancer.gov/</a></p><p id="P55">breast cancer | colorectal cancer | lung cancer</p><p id="P56"><strong>LocusLink</strong>: <a href="http://www.ncbi.nlm.nih.gov/LocusLink/" ref="reftype=extlink&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=nihms" target="pmc_ext">http://www.ncbi.nlm.nih.gov/LocusLink/</a></p><p id="P57"><em>APC</em> | <em>ATM</em> | <em>BMPR1A</em> | <em>BRCA1</em> | <em>BRCA2</em> | <em>CDK4</em> | <em>CDKN2A</em> | <em>CHK2</em> | CREBBP | EGFR | <em>EP300</em> | <em>ETV6</em> | <em>FHIT</em> | <em>FLT3</em> | HRAS | <em>KIT</em> | MET | <em>MLH1</em> | <em>MLL</em> | <em>NTR3</em> | <em>NTRK1</em> | PAX8 | <em>PDGFRA</em> | <em>PPAR&#x003b3;</em> | <em>PRCC</em> | PRKAR1A | <em>PTEN</em> | <em>RET</em> | <em>RUNXBP2</em> | <em>STK11</em> | <em>TFE3</em> | TGF-&#x003b2; | <em>TGF-&#x003b2;RII</em> | <em>TP53</em> | <em>WWOX</em></p><p id="P58"><strong>OMIM</strong>: <a href="http://www.ncbi.nlm.nih.gov/omim/" ref="reftype=extlink&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=nihms" target="pmc_ext">http://www.ncbi.nlm.nih.gov/omim/</a></p><p id="P59">Carney complex | Peutz&#x02013;Jegher syndrome</p></div><div id="APP2" class="sec sec-last"><h3>FURTHER INFORMATION</h3><p id="P60"><strong>Cancer Genome Project</strong>: <a href="http://www.sanger.ac.uk/CGP/" ref="reftype=extlink&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=nihms" target="pmc_ext">http://www.sanger.ac.uk/CGP/</a></p><p id="P61"><strong>Ensembl Human Genome Browser web site</strong>: <a href="http://www.ensembl.org/Homo_sapiens/" ref="reftype=extlink&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=nihms" target="pmc_ext">http://www.ensembl.org/Homo_sapiens/</a></p><p id="P62"><strong>Pfam home page</strong>: <a href="http://www.sanger.ac.uk/Software/Pfam/" ref="reftype=extlink&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=nihms" target="pmc_ext">http://www.sanger.ac.uk/Software/Pfam/</a></p><p id="P63"><strong>Wellcome Trust Sanger Institute web site</strong>: <a href="http://www.sanger.ac.uk/" ref="reftype=extlink&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=nihms" target="pmc_ext">http://www.sanger.ac.uk/</a></p><p id="P64">Access to this interactive links box is free online.</p></div></div></div><div id="__fn-groupid1060478" class="tsec sec"><h2 class="head no_bottom_margin" id="__fn-groupid1060478title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"><p id="P52" class="p p-first-last">Competing interests statement The authors declare that they have no competing financial interests.</p></p></div></div><div id="__ref-listid1042008" class="tsec sec"><h2 class="head no_bottom_margin" id="__ref-listid1042008title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Reddy EP, Reynolds RK, Santos E, Barbacid M. A point mutation is responsible for the acquisition of transforming properties by the T24 human bladder carcinoma oncogene. <span><span class="ref-journal">Nature. </span>1982;<span class="ref-vol">300</span>:149–152.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/7133135" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Tabin CJ, et al.  Mechanism of activation of a human oncogene. <span><span class="ref-journal">Nature. </span>1982;<span class="ref-vol">300</span>:143–149.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/6290897" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span><br /><span> [References 1 and 2 are seminal papers that defined the existence of oncogene sequences in cancer-cell genomes and that first identified a mutation involved in human cancer.]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Parsons R, et al.  Hypermutability and mismatch repair deficiency in RER+ tumor cells. <span><span class="ref-journal">Cell. </span>1993;<span class="ref-vol">75</span>:1227–1236.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/8261516" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Fishel R, et al.  The human mutator gene homolog <em>MSH2</em> and its association with hereditary nonpolyposis colon cancer. <span><span class="ref-journal">Cell. </span>1993;<span class="ref-vol">75</span>:1027–1038.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/8252616" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Leach FS, et al.  Mutations of a mutS homolog in hereditary nonpolyposis colorectal cancer. <span><span class="ref-journal">Cell. </span>1993;<span class="ref-vol">75</span>:1215–1225.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/8261515" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Merlo A, et al.  5&#x02032; CpG island methylation is associated with transcriptional silencing of the tumour suppressor <em>p16</em>/<em>CDKN2</em>/<em>MTS1</em> in human cancers. <span><span class="ref-journal">Nature Med. </span>1995;<span class="ref-vol">1</span>:686–692.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/7585152" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Kane MF, et al.  Methylation of the <em>hMLH1</em> promoter correlates with lack of expression of hMLH1 in sporadic colon tumors and mismatch repair-defective human tumor cell lines. <span><span class="ref-journal">Cancer Res. </span>1997;<span class="ref-vol">57</span>:808–811.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/9041175" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Baylin S, Bestor TH. Altered methylation patterns in cancer cell genomes: cause or consequence? <span><span class="ref-journal">Cancer Cell. </span>2002;<span class="ref-vol">1</span>:299–305.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/12086841" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Duval A, Hamelin R. Mutations at coding repeat sequences in mismatch repair-deficient human cancers: toward a new concept of target genes for instability. <span><span class="ref-journal">Cancer Res. </span>2002;<span class="ref-vol">62</span>:2447–2454.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/11980631" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Duval A, et al.  Evolution of instability at coding and non-coding repeat sequences in human MSI-H colorectal cancers. <span><span class="ref-journal">Hum. Mol. Genet. </span>2001;<span class="ref-vol">10</span>:513–518.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/11181575" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Grady WM, et al.  Mutational inactivation of transforming growth factor &#x003b2; receptor type II in microsatellite stable colon cancers. <span><span class="ref-journal">Cancer Res. </span>1999;<span class="ref-vol">59</span>:320–324.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/9927040" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Ohta M, et al.  The <em>FHIT</em> gene, spanning the chromosome 3p14.2 fragile site and renal carcinoma-associated t(3;8) breakpoint, is abnormal in digestive tract cancers. <span><span class="ref-journal">Cell. </span>1996;<span class="ref-vol">84</span>:587–597.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/8598045" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Bednarek AK, et al.  WWOX, a novel WW domain-containing protein mapping to human chromosome 16q23.3-24.1, a region frequently affected in breast cancer. <span><span class="ref-journal">Cancer Res. </span>2000;<span class="ref-vol">60</span>:2140–2145.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/10786676" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Sutherland GR, Baker E, Richards RI. Fragile sites still breaking. <span><span class="ref-journal">Trends Genet. </span>1998;<span class="ref-vol">14</span>:501–506.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/9865156" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Kamb A, et al.  A cell cycle regulator potentially involved in genesis of many tumor types. <span><span class="ref-journal">Science. </span>1994;<span class="ref-vol">264</span>:436–440.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/8153634" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Li J, et al.  <em>PTEN</em>, a putative protein tyrosine phosphatase gene mutated in human brain, breast, and prostate cancer. <span><span class="ref-journal">Science. </span>1997;<span class="ref-vol">275</span>:1943–1946.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/9072974" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Laken SJ, et al.  Familial colorectal cancer in Ashkenazim due to a hypermutable tract in <em>APC</em>. <span><span class="ref-journal">Nature Genet. </span>1997;<span class="ref-vol">17</span>:79–83.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/9288102" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Meijers-Heijboer H, et al.  Low-penetrance susceptibility to breast cancer due to CHEK2(*)1100delC in noncarriers of <em>BRCA1</em> or <em>BRCA2</em> mutations. <span><span class="ref-journal">Nature Genet. </span>2002;<span class="ref-vol">31</span>:55–59.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/11967536" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Shih C, Shilo BZ, Goldfarb MP, Dannenberg A, Weinberg RA. Passage of phenotypes of chemically transformed cells via transfection of DNA and chromatin. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>1979;<span class="ref-vol">76</span>:5714–5718.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC411720/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/230490" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Krontiris TG, Cooper GM. Transforming activity of human tumor DNAs. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>1981;<span class="ref-vol">78</span>:1181–1184.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC319971/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/6940134" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Baker SJ, et al.  p53 gene mutations occur in combination with 17p allelic deletions as late events in colorectal tumorigenesis. <span><span class="ref-journal">Cancer Res. </span>1990;<span class="ref-vol">50</span>:7717–7722.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/2253215" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Hwang S-J, Lozano G, Amos CI, Strong LC. Germline p53 mutations in a cohort with childhood sarcoma: sex differences in cancer risk. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2003;<span class="ref-vol">72</span>:975–983.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC1180359/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/12610779" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Hemminki A, et al.  A serine/threonine kinase gene defective in Peutz&#x02013;Jeghers syndrome. <span><span class="ref-journal">Nature. </span>1998;<span class="ref-vol">391</span>:184–187.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/9428765" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Sanchez-Cespedes M, et al.  Inactivation of LKB1/STK11 is a common event in adenocarcinomas of the lung. <span><span class="ref-journal">Cancer Res. </span>2002;<span class="ref-vol">62</span>:3659–3662.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/12097271" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Futreal PA, et al.  <em>BRCA1</em> mutations in primary breast and ovarian carcinomas. <span><span class="ref-journal">Science. </span>1994;<span class="ref-vol">266</span>:120–122.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/7939630" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Lancaster JM, et al.  <em>BRCA2</em> mutations in primary breast and ovarian cancers. <span><span class="ref-journal">Nature Genet. </span>1996;<span class="ref-vol">13</span>:238–240.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/8640235" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Nikiforov YE. RET/PTC rearrangement in thyroid tumors. <span><span class="ref-journal">Endocr. Pathol. </span>2002;<span class="ref-vol">13</span>:3–16.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/12114746" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Kroll TG, et al.  <em>PAX</em>8&#x02013;<em>PPAR&#x003b3;</em>1 fusion in oncogene human thyroid carcinoma. <span><span class="ref-journal">Science. </span>2000;<span class="ref-vol">289</span>:1357–1360.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/10958784" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Sidhar SK, et al.  The t(X;1)(p11. 2;q21. 2) translocation in papillary renal cell carcinoma fuses a novel gene <em>PRCC</em> to the <em>TFE3</em> transcription factor gene. <span><span class="ref-journal">Hum. Molec. Genet. </span>1996;<span class="ref-vol">5</span>:1333–1338.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/8872474" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Tognon C, et al.  Expression of the <em>ETV6</em>&#x02013;<em>NTRK3</em> gene fusion as a primary event in human secretory breast carcinoma. <span><span class="ref-journal">Cancer Cell. </span>2002;<span class="ref-vol">2</span>:367–376.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/12450792" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Ayton PM, Cleary ML. Molecular mechanisms of leukemogenesis mediated by MLL fusion proteins. <span><span class="ref-journal">Oncogene. </span>2001;<span class="ref-vol">20</span>:5695–5707.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/11607819" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Eng C, Mulligan LM. Mutations of the <em>RET</em> proto-oncogene in the multiple endocrine neoplasia type 2 syndromes, related sporadic tumours, and Hirschsprung disease. <span><span class="ref-journal">Hum. Mutat. </span>1997;<span class="ref-vol">9</span>:97–109.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/9067749" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Wong AJ, et al.  Structural alterations of the epidermal growth factor receptor gene in human gliomas. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>1992;<span class="ref-vol">89</span>:2965–2969.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC48784/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/1557402" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Zhuang Z, et al.  Trisomy 7-harbouring non-random duplication of the mutant <em>MET</em> allele in hereditary papillary renal carcinomas. <span><span class="ref-journal">Nature Genet. </span>1998;<span class="ref-vol">20</span>:66–69.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/9731534" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Bongarzone I, et al.  Molecular characterization of a thyroid tumor-specific transforming sequence formed by the fusion of ret tyrosine kinase and the regulatory subunit RI&#x003b1; of cyclic AMP-dependent protein kinase A. <span><span class="ref-journal">Molec. Cell. Biol. </span>1993;<span class="ref-vol">13</span>:358–366.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC358915/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/7678053" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Kirschner LS, et al.  Mutations of the gene encoding the protein kinase A type I-&#x003b1; regulatory subunit in patients with the Carney complex. <span><span class="ref-journal">Nature Genet. </span>2000;<span class="ref-vol">26</span>:89–92.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/10973256" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Bateman A, et al.  The Pfam protein families database. <span><span class="ref-journal">Nucleic Acids Res. </span>2002;<span class="ref-vol">30</span>:276–280.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="/pmc/articles/PMC99071/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/11752314" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Savitsky K, et al.  A single ataxia telangiectasia gene with a product similar to PI-3 kinase. <span><span class="ref-journal">Science. </span>1995;<span class="ref-vol">268</span>:1749–1753.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/7792600" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Howe JR, et al.  Germline mutations of the gene encoding bone morphogenetic protein receptor 1A in juvenile polyposis. <span><span class="ref-journal">Nature Genet. </span>2001;<span class="ref-vol">28</span>:184–187.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/11381269" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Gilliland DG, Griffin JD. The roles of FLT3 in hematopoiesis and leukemia. <span><span class="ref-journal">Blood. </span>2002;<span class="ref-vol">100</span>:1532–1542.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/12176867" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Hirota S, et al.  Gain-of-function mutations of c-kit in human gastrointestinal stromal tumors. <span><span class="ref-journal">Science. </span>1998;<span class="ref-vol">279</span>:577–580.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/9438854" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Heinrich MC, et al.  PDGFRA activating mutations in gastrointestinal stromal tumors. <span><span class="ref-journal">Science. </span>2003;<span class="ref-vol">299</span>:708–710.</span>  <span class="nowrap ref pubmed">[<a href="/pubmed/12522257" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2665285&amp;issue-id=177762&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=nihms">PubMed</a>]</span></span></div></div></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2665285/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2665285/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2665285/pdf/ukmss-2560.pdf">PDF (801K)</a></li> | <li><a href="#" data-citationid="PMC2665285" class="citationexporter ctxp">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2665285%2F"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4144485/img/4047626" alt="Share on Facebook" />
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2665285%2F&amp;text=A%20CENSUS%20OF%20HUMAN%20CANCER%20GENES"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4144485/img/4047627" alt="Share on Twitter" />
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2665285%2F"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4144485/img/4047628" alt="Share on Google Plus" />
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="14993899" data-aiid="2665285" data-aid="2665285" data-iid="177762" data-domainid="358" data-domain="wtpa" data-accid="PMC2665285" data-md5="bb62b9b5a82b2002d52fc59b6ed3265f"></div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="">Support Center</a>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2016-12-03T23:07:32-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.43&amp;Host=ptpmc101&amp;ncbi_phid=8A1A20C684395CE100000000004D0045&amp;ncbi_session=8A1A20C684396831_0077SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2665285%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        </div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2665285/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- 8A1A20C684396831_0077SID /projects/PMC/PMCViewer@4.43 ptpmc101 v4.1.r519325 Tue, Nov 15 2016 02:37:57 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4144485/js/3879255/4121861/3818874/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script></body>
</html>